setup.py

```diff
@@ -0,0 +1,15 @@
+from importlib.metadata import entry_points
```

Comment:
```suggestion
```
Doesn't seem necessary, entry_points is just a name of the parameter in the function below

Reply:
adding changes